src/PhpSpec/Locator/PSR0/PSR0Locator.php

```diff
@@ -63,14 +63,14 @@ class PSR0Locator implements ResourceLocatorInterface
      * @param string     $psr4Prefix
      */
     public function __construct(
+        Filesystem $filesystem,
```

Comment:
Moving a parameter around breaks BC. this may be fine, as this class is mostly internal, but it should be acknowledged.
@ciaranmcnulty it may be great to add ``@internal`` annotations where appropriate